### PR TITLE
refactor(rxButton): FRMW-735 convert Buttons + rxButton to Element

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-735

### LGTMs
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `src/atoms/rxButton` and `demo/templates/atoms/buttons.html` to `src/elements/Buttons` Element.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Button` and `rxButton` Element

### Demo Page
buttons - encoreui 2016-04-12 13-27-31

### API Page
rxbutton 2016-04-12 13-28-29

File: grunt-tasks/options/html2js.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ var srcDirs = [
     'src/atoms',
     'src/components',
     'src/molecules',
-    'src/quarks'
+    'src/quarks',
```
Comment (CITguy) [inline grunt-tasks/options/html2js.js]: You can remove `src/quarks`. That directory is no longer present.

File: src/elements/Buttons/Buttons.meta.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "displayName": "Buttons",
+    "stability": "unstable",
+    "description": "Several examples and styles of Buttons available for different contexts.",
+    "api": "directive:rxButton",
+    "keywords": [
+        "button",
+        "rxButton",
```
Comment (CITguy) [inline src/elements/Buttons/Buttons.meta.json]: Please alphabetize keywords.

**Comment by 100stacks** [comment]
@CITguy, PR updated with comments.

**Comment by CITguy** [comment]
LGTM

**Comment by glynnis** [comment]
LGTM

**Pull request merged**